docs/guide/content/installation.md

```diff
@@ -1,12 +1,89 @@
 # Installation Guide
 
-## Launch OnDemand Loop from Dataverse
+This guide explains how to build and deploy OnDemand Loop alongside an existing
+Open OnDemand installation. The application is designed to run as a Passenger
+app so that it lives under `/var/www/ood/apps/sys/loop` in the same manner as
+the standard Dashboard app.
 
-Dataverse supports **External Tools**, which enable integrations with external web applications. This section explains how to integrate **Dataverse with OnDemand Loop**, so that users can launch Loop directly from a dataset page. This allows researchers to initiate file transfers from Dataverse into an HPC cluster via the OnDemand Loop interface.
+## System Requirements
 
-> Note: This complements the existing Loop-side integration, where users can browse Dataverses from within the Loop app. Here, the workflow begins in Dataverse and hands off to Loop.
+OnDemand Loop targets the same software versions used by Open OnDemand itself.
+Use the versions below (or the versions bundled with your OOD release):
 
-With Dataverse running at `http://localhost:8080` and the Loop app accessible at `https://localhost:33000`, register the external tool with the following command:
+- **Ruby** 3.1
+- **Node.js** 18
+- **Open OnDemand** 3.1 or newer
+
+## Building the Application
+
+The repository includes a Makefile with helper targets. The preferred way to
+compile Loop is to install all Ruby gems and Node packages into local folders,
+avoiding conflicts with system-wide installs. This is exactly how the
+`scripts/loop_build.sh` script operates and how the application has been tested.
+See that script for the exact commands executed during the build.
+To run the build inside the Docker based builder image, execute:
+
+```bash
+make release_build
+```
+
+The script stores Ruby gems under `vendor/bundle` and Node packages in
+`node_modules` within the `application` directory so that the build is isolated
+from system packages.
+
+This command installs all Ruby and Node dependencies and precompiles the CSS and
+JavaScript assets. The same steps can be performed manually by executing the
+`scripts/loop_build.sh` script on a machine that has Ruby and Node available.
+
+## Deployment Options
+
+### 1. Build on the Server
+
+Clone the repository (or a release tag) directly into the OOD `sys` directory
+and run the build script:
+
+```bash
+cd /var/www/ood/apps/sys
+git clone <repo-or-release> loop
+cd loop
+./scripts/loop_build.sh
+```
+
+### 2. Build Elsewhere and Copy
+
+If you prefer to compile the application in a controlled environment, run the
+build steps on another machine and then copy `ondemand-loop/application` directory to
+`/var/www/ood/apps/sys` on your production server. Then rename it to `loop`, ie: `/var/www/ood/apps/sys/loop`.
+Ensure file permissions are preserved during the transfer.
+
```

Comment:
Here we have two different file configurations:

In (1) Build on the server the application source code will be located in `/var/www/ood/apps/sys/loop/application` whereas in (2) Build Elsewhere and copy, the source code will be located in `/var/www/ood/apps/sys/loop` . Maybe in (1) we need to do something like this:

```bash
cd
git clone <repo-or-release> loop
cd loop
./scripts/loop_build.sh
cp -R application /var/www/ood/apps/sys/loop
```

Reply:
I yes - need to fix that